### ISSUE TITLE
Bug in static mode

### ISSUE THREAD

**Issue by KillianBabilotte** [verificarlo/verificarlo#129]
I tried to install verificarlo on my computer following the README.md instruction. After running the installcheck I've got one test failure on the "test_static_mode". The file test.log said : 

```
".vfcwrapper.o : Dans la fonction « vfc_init » :
/usr/local/bin/../include/vfcwrapper.c:(.text+0x210): avertissement : Using 'dlopen' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
test: Cannot load backend libinterflop_ieee.so: dlopen error
libinterflop_ieee.so: cannot open shared object file: No such file or directory "
```

**Comment by yohanchatelain** [comment]
@KillianBabilotte are your sure that is not a problem of LD_LIBRARY_PATH? Because I can't reproduce your bug.

**Comment by KillianBabilotte** [comment]
I'm not sure about the problem cause because it disappear on the following executions and I can't reproduce it either. But between the failed execution and the succeed one, I changed absolutely nothing, I only rebooted my machine. So I kept the Issue opened because I found it even more strange. I called it a failure because it is the return output in the terminal in the list of test execution by command make installcheck.
